src/TypeSpec.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -40,6 +40,18 @@ export const $lib = createTypeSpecLibrary({
             messages: {
                 default: paramMessage`No Route for service for service ${"service"}`
             }
```

Comment:
I didn't find the change of package.json, so please update TCGC to the latest version.